content/apps/using-openid.md

```diff
@@ -119,13 +118,13 @@ The ID Token sent by Akeneo PIM contains a signature, and you must verify it to
 tampered with.
 
 To validate the signature, you must retrieve the public key available at the
-url `{PIM}/connect/apps/v1/openid/public-key`.
+URL `{PIM}/connect/apps/v1/openid/public-key`.
 
 Then, follow the instructions of the library you are using.
 
 ::: warning
-The pair of private/public keys is regenerated regularly for security reasons. You should always retrieve
-the lastest public key when validating a signature.
+The pair of private/public keys are regenerated regularly for security reasons. It would be best if you always retrieved 
```

Comment:
Did you drop "should" to make the instruction more lenient?
To keep the tone formal, we could use "It is recommended to retrieve...". But I don't mind either way.

Reply:
I'm not really fan of the changes proposed in this PR.
It's only changes proposed automatically by the grammarly tool.